website/pages/en/index.js

```diff
@@ -58,35 +62,21 @@ const Logo = props => (
 
 const ProjectTitle = props => (
   <h2 className="projectTitle">
-    {siteConfig.title}
-    <small>{siteConfig.tagline}</small>
```

Comment:
I preferred the older ProjectTitle, the font change and logo are too much IMO

<img width="286" alt="screen shot 2018-08-24 at 17 50 58" src="https://user-images.githubusercontent.com/1408093/44594376-4c48c200-a7c6-11e8-9677-4e64fc79c1db.png">

I don't think it's necessary to customize this.